data-model/fdo-profiles/0.1.0/digital-specimen/schema/digital-specimen-request.json

```diff
@@ -0,0 +1,234 @@
+{
+  "$id": "ttps://schemas.dissco.tech/schemas/fdo-profiles/0.1.0/digital-specimen-request.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "description": "FDO Profile for Digital Specimens",
+  "$comment":"FDO Profile Version 0.1.0",
+  "allof": [
+    {
+      "$ref": "https://schemas.dissco.tech/schemas/fdo-profiles/0.1.0/handle-request.json"
+    },
+    {
+      "$ref": "https://schemas.dissco.tech/schemas/fdo-profiles/0.1.0/doi-kernel/schema/doi-request.json"
+    }
+  ],
+  "properties": {
+    "specimenHost": {
+      "type": "string",
+      "description": "ROR or, in absence of a ROR, Wikidata Qnumber for the administrative organisation hosting the specimen. Same as: Latimer Core identifierSource.",
+      "example": "https://ror.org/0566bfb96",
+      "$comment": "idx = 200"
+    },
+    "specimenHostName": {
+      "type": "string",
+      "description": "Name for the administrative organisation as derived from ROR or Wikidata at the point of creating the FDO record.",
+      "example": "Naturalis Biodiversity Center",
+      "$comment": "idx = 201"
+    },
+    "primarySpecimenObjectId": {
+      "type": "string",
+      "description": "Primary local identifier used to identify the physical specimen, which is a preserved material entity. This identifier is usually physically attached to the specimen, e.g. as barcode. It is recommended to use a global and resolvable identifier if available. The host of the physical specimen should be able to find the physical specimen with only this ID plus, if that is not globally unique, the primarySpecimenObjectIdName",
+      "example": "RMNH.1.2b",
+      "$comment": "idx = 202"
+    },
+    "primarySpecimenObjectIdType": {
+      "enum": [
+        "Global",
+        "Resolvable",
+        "Local"
+      ],
+      "description": "Nature of institutional identifier",
+      "$comment": "idx = 203"
+    },
+    "normalisedSpecimenObjectId": {
+      "type": "string",
+      "description": "For internal processing purposes only to make a local identifier globally unique. This is the primarySpecimenObjectId if it is globally unique or a combination of ROR ID string, source system ID string, primarySpecimenObjectId if it is a proprietary identifier.",
+      "example": "0566bfb96:20.5000.1025/GEE-W3J-HL2:RMNH.1.2b",
+      "$comment": "idx = 205"
+    },
+    "specimenObjectIdAbsenceReason": {
+      "type": "string",
+      "description": "*Not currently supported.* Either this attribute or primarySpecimenObjectId needs to be filled, if both are filled the FDO record is invalid. Max 255 chars. Note that absence of the ID poses a challenge on avoiding duplicate digital specimen IDs.",
```

Comment:
That's a good idea, but I'm hesitant to implement it. 2 issues:
1. We don't have a workflow defined for when a specimen doesn't have a specimen object id. The handle API (which is the primary use-case for this schema right now) would crash. 
2. We use a tool called [jsonschema2pojo](https://github.com/DiSSCo/openDS/tree/master/data-model/fdo-profiles) to create java objects from json schemas. Unfortunately, jsonschema2pojo doesn't currently support the "oneOf" element, and would negatively affect the java objects derived from the schema. 

It would be better practice to use the oneOf element here, but until at least issue #1 is resolved and #2 has a decent workaround, I don't want to implement this element. 

PS Check out [schemas.dissco.tech](https://schemas.dissco.tech/), it serves the fdo profiles in this repo as jsons to the applications that need them :sunglasses: